packages/@okta/vuepress-site/docs/guides/pwd-optional-new-sign-up-email/main/java/integrationsteps.md

```diff
@@ -86,6 +90,11 @@ If `true` &mdash; and all the listed authenticators are optional &mdash; add a *
 
 ![A form showing a list of authenticators to enroll, and Submit, Cancel, and Skip buttons.](/img/pwd-optional/pwd-optional-sign-up-authenticators-page.png)
 
+<!--
+
+Source image: https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%93%8A-Updated-Diagrams-for-Dev-Docs?node-id=1978%3A1741 pwd-optional-sign-up-authenticators-page
```

Comment:
the name pwd-optional-sign-up-authenticators-page has the word "sign-up"

Reply:
Yes it does. It shows the authenticators page in the signup flow. Isn't that what you intended?